docs/modules/README.md

```diff
@@ -19,3 +19,4 @@ ESLint plugin.
 - [`MaterializeCSS`](./materialize-css.md)
 - [`moment.js`](./momentjs.md)
 - [`npm-run-all`](./npm-run-all.md)
+- [`cpx`](./cpx.md)
```

Comment:
tiny nit: can you keep this list a-z? just need to move it to the top

Reply:
Fixed in [091dad5](https://github.com/es-tooling/module-replacements/pull/19/commits/091dad584c8600b7375181f4434de41b85b88c8e)